bin/post_compile

```diff
@@ -5,23 +5,16 @@
 set -euo pipefail
 
 # Make the current Git revision accessible at <site-root>/revision.txt
-echo "$SOURCE_VERSION" > ui/revision.txt
-
-# Create a `dist/` directory containing built/minified versions of the `ui/` assets.
-# Uses the node binaries/packages installed by the nodejs buildpack previously.
-yarn build
```

Comment:
The `yarn build` in `bin/post_compile` has now been moved to the `heroku-prebuild` `scripts` entry in `package.json` instead, since it makes sense to keep the JS things together. See:
https://devcenter.heroku.com/articles/nodejs-support#heroku-specific-build-steps